packages/syft/src/syft/core/tensor/functions.py

```diff
@@ -27,13 +27,13 @@ def mean(
 
 @implements(Tensor, np.max)
 def npmax(*args: List[Any], **kwargs: Dict[Any, Any]) -> Tensor:
```

Comment:
*args is variadic so Tuple[Any, ...] is probably more correct.

Reply:
fixed